src/Explorer/Controls/Editor/EditorComponent.ts

```diff
@@ -52,12 +52,14 @@ class EditorViewModel extends JsonEditorViewModel {
   protected registerCompletionItemProvider() {
     let sqlCompletionItemProvider = new SqlCompletionItemProvider();
     if (EditorViewModel.providerRegistered.indexOf("sql") < 0) {
-      monaco.languages.registerCompletionItemProvider("sql", sqlCompletionItemProvider);
+      // TODO Remove when @azure/cosmos-language-service is updated

```

Comment:
I have a [PR open](https://github.com/Azure/cosmos-sql-language-service/pull/5) to fix this.